src/test/java/org/sagebionetworks/bridge/models/files/FileReferenceTest.java

```diff
@@ -0,0 +1,32 @@
+package org.sagebionetworks.bridge.models.files;
+
+import static org.sagebionetworks.bridge.TestConstants.GUID;
+import static org.sagebionetworks.bridge.TestConstants.TIMESTAMP;
+import static org.sagebionetworks.bridge.config.Environment.PROD;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+import org.testng.annotations.Test;
+
+import org.sagebionetworks.bridge.config.BridgeConfigFactory;
+
+public class FileReferenceTest {
```

Comment:
What about the null guid case and the null createdOn case?

We also need an equalsVerifier test and a JSON serialization test.

Reply:
Added tests that nulls don't break anything and added equalsVerifier.